### PR TITLE
Align PHP require version to composer.json

### PR DESCRIPTION
**Pull request by rawdreeg**
### Changes proposed in this Pull Request:

This PR make changes to align PHP require version to composer.json

- [x] Do the changed files pass `phpcs` checks? Please remove `phpcs:ignore` comments in changed files and fix any issues, or delete if not practical.

### Changelog entry

>Tweak - Align PHP require version to composer.json

### CODE REVIEW

File: facebook-for-woocommerce.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * Author: Facebook
  * Author URI: https://www.facebook.com/
  * Version: 3.2.5
- * Requires at least: 5.6
+ * Requires at least: 7.4
```
Comment (ibndawood) [inline facebook-for-woocommerce.php:15]: This is the WordPress version. I am setting the same as the one used in WooCommerce..
```suggestion
 * Requires at least: 6.4
 * Requires PHP: 7.4
```

**Review by ibndawood — CHANGES_REQUESTED** [review]
Thank you, @rawdreeg. I've made some changes. Here is the plugin header details: https://developer.wordpress.org/plugins/plugin-basics/header-requirements/

**Comment by rawdreeg** [comment]
Thanks @ibndawood  , for the Requires at least, shouldn't that be 5.6? then we can update readme as well.

**Comment by ibndawood** [comment]
I just used the same value as in WooCommerce core here: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/woocommerce.php

**Review by ibndawood — APPROVED** [review]

**Comment by rawdreeg** [comment]
> I just used the same value as in WooCommerce core here: [woocommerce/woocommerce@trunk/plugins/woocommerce/woocommerce.php](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/woocommerce.php?rgh-link-date=2024-07-11T13%3A34%3A27Z)

Ah neat. We can make another PR to adjust the minimum required WP.

**Pull request merged**